pkg/operator/staticpod/controller/installer/installer_controller.go

```diff
@@ -1087,21 +1138,17 @@ func (c InstallerController) Sync(ctx context.Context, syncCtx factory.SyncConte
 
 	// Update failing condition
 	// If required certs are missing, this will report degraded as we can't create installer pods because of this pre-condition.
-	cond := operatorv1.OperatorCondition{
-		Type:   condition.InstallerControllerDegradedConditionType,
-		Status: operatorv1.ConditionFalse,
-	}
-	if err != nil {
-		cond.Status = operatorv1.ConditionTrue
-		cond.Reason = "Error"
-		cond.Message = err.Error()
+	nodeStatusApplyConfigurations := prepareNodeStatusApplyConfigurationForUpdatedNode(originalOperatorStatus.NodeStatuses, updatedNode)
```

Comment:
Does this mean that this controller will take ownership of NodeStatuses that were originally from other fieldManagers?